xarray/backends/scipy_.py

```diff
@@ -64,7 +64,13 @@ def get_variable(self, needs_lock=True):
         return ds.variables[self.variable_name]
 
     def __getitem__(self, key):
-        data = NumpyIndexingAdapter(self.get_variable().data)[key]
+        indexable = NumpyIndexingAdapter(self.get_variable().data)
```

Comment:
I might try with `data = explicit_indexing_adapter(NumpyIndexingAdapter(...), indexing_support=indexing.VECTORIZED)` (or something like that). to follow the other backends. And the implement this dispatching logic in a backwards-compatible manner in `explicit_indexing_adapter` as discussed in my proposal

Reply:
the latest commit brings the `scipy` backend up to the same level as the other backends by using the `indexing.explicit_indexing_adapter()` function.